hashgraph-transfer.md

```diff
@@ -9,37 +9,84 @@ When transferring ownership of a repository from [Hashgraph](https://github.com/
+  - Appropriate service account is configured in [Hiero-Ledger][hiero]
+  - List of currrently published packages (from [Hashgraph][hashgraph]) is provided to **Repository Transfer Agent**
+- If codecov artifacts are captured for the repository
+  - **Repository Transfer Agent** is made aware
+
+- The [@hashgraph/devops-ci team][devops-ci] will capture the repository variables and variable states and provide to
+  the TA prior to day of transfer
+- The [@hashgraph/devops-ci team][devops-ci] will capture the repository secrets and provide this list to the TA prior
+  to day of transfer
+
+## Day Before Checks
+
+The day before a repository is transferred the TA and AA shall verify the following checks have been complete
+
+- DCO Audit is complete
+- License Audit is complete
+- Pre-Transfer Checks are complete
+  - TA has the following:
+    - New repository Information
+    - List of MC packages
+    - List of Gradle Plugin Central packages
+    - List of NPMJS packages
+    - Awareness of Codecov needs
+    - List of repository variables/states
+    - List of repository secrets
+    - Repository Point of Contact for Day of questions
+- Developer Experience team is allerted that the repository is ready to transfer
+
+The Developer Experience Team shall announce the upcoming repository transfer to the community once the TA and AA verify
+the repository is ready for transfer
 
 ## Transfer Day
 
 Throughout the transfer process both TA and AA shall be available.
 
 The following steps shall occur to transfer a repository into Hiero from a source organization
 
-- The TA will generate a pull request in the [Hiero-Ledger Governance](https://github.com/hiero-ledger/governance) repository
-  - The TA will modify the `config.yaml` file in this repository with the following information
-    - The new maintainer group name (`REPO-maintainers`)
-    - The new maintainers for the group
-    - The new committer group name (`REPO-committers`)
-    - the new set of committers for that group
-    - The new repository to pull in
-    - The `tsc` group should have the `admin` role on the repository
-    - The `github-maintainers` group should have the `admin` role on the repository
-    - The `github-committers` group should have the `write` role on the repository
-    - The `REPO-maintainers` group should have the `maintain` role on the repository
-    - The `REPO-committers` group should have the `write` role on the repository
-- The AA will approve the changes in `Hiero-Ledger/governance`
+- The TA will open a new ticket in the [Governance][governance] repository
+  - The TA will create a new branch on the [Governance][governance] repository's ticket
+- The TA will modify the `config.yaml` file in this repository with the following information
+  - The new maintainer group name (`REPO-maintainers`)
+  - The new maintainers for the group
+  - The new committer group name (`REPO-committers`)
+  - the new set of committers for that group
```

Comment:
Capitalize `The` (the rest of the list has caps, this one does not)

Reply:
```suggestion
  - The new set of committers for that group
```